### ISSUE TITLE
Cannot tab through recipe cards themselves in the main view.

### ISSUE THREAD

**Issue by Codeherder19** [deadbelly/whats-cookin#48]
Cannot tab through card elements in main view, yet everything else seems to be able to be tabbed through and selected. Screen reader reads each element to the user successfully but does not read the recipe cards.
